packages/endpoints-organizations-registrar/src/entities/organizations/orchestrators/build-non-custodied-organization.js

```diff
@@ -70,6 +70,7 @@ const buildNonCustodiedOrganization = async (
       normalizedProfileName: normalizeProfileName(profile.name),
       didNotCustodied: true,
       services: newServices,
+      invitationId: invitation?._id ?? undefined,
```

Comment:
@nassan im not sure adding `invitationId` to organzation is is necessary any longer. You can instead have a helper function that takes the organization, iterates through the services and returns the first  `invitationId`